docs/resources/vpc_peering.md

```diff
@@ -6,7 +6,7 @@ description: |-
 ---
 
 ### Resource: `instaclustr_vpc_peering`  
-A resource for managing VPC peering connections on Instaclustr Managed Platform. This is only avaliable for clusters hosted with the AWS provider.  
+A resource for managing VPC peering connections on Instaclustr Managed Platform. This is avaliable for clusters hosted with the AWS provider and the GCP provider.
```

Comment:
I'd reword this to be `This is available for clusters hosted with AWS and GCP.` - typo in word available and cleaning up provider names